db/migrate/20200701212803_add_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTokenToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :token, :string
+  end
```

Comment:
nice! we may need to rename this if we end up with more then one token type per user.